server/src/db/models/user.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    id: { type: DataTypes.STRING, primaryKey: true },
+    email: { type: DataTypes.STRING, unique: true },
+    password: DataTypes.STRING,
+    createdAt: DataTypes.DATE,
+    updatedAt: DataTypes.DATE,
+  });
```

Comment:
you may want to decided whether or not you want to add `freezeTableName` to the definition of this model

Reply:
decided to go with plural table names